addons/csi-vmware-cloud-director/vcloud-csi-config.yaml

```diff
@@ -5,12 +5,13 @@ metadata:
   name: vcloud-csi-configmap
   namespace: kube-system
 data:
+  # host shouldn't have the `/api` suffix.
   vcloud-csi-config.yaml: |+
     vcd:
-      host: {{ required "Please provide VCD_URL" .Credentials.VCD_URL }}
+      host: {{ required "Please provide VCD_URL" (trimSuffix "/api" .Credentials.VCD_URL) }}
       org: {{ required "Please provide VCD_ORG" .Credentials.VCD_ORG }}
       vdc: {{ required "Please provide VCD_VDC" .Credentials.VCD_VDC }}
-      vAppName: {{ default .Config.Name .Params.vApp }}
+      vAppName: {{ default .Config.CloudProvider.VMwareCloudDirector.VApp .Params.vApp }}
```

Comment:
it seems like we will have `VMwareCloudDirector.VApp` ether way, so having `default` here doesn't make much sense, together with `.Params.vApp`